include/open62541/server_pubsub.h

```diff
@@ -287,6 +288,7 @@ typedef struct {
 typedef struct {
     UA_String name;
     UA_PublishedDataSetType publishedDataSetType;
+    UA_Boolean sendViaWriterGroupTopic;
```

Comment:
Is the insertion of the field necessary?
And what's the field for?

Reply:
The field allows for selective batching. DataSetMessages of DataSetWriters can be batched by their common WriterGroup using the `UA_BrokerDataSetWriterTransportDataType` of the WriterGroup according to Part 14. PublishedDataSets of Writers grouped by a common WriterGroup with this flag set will be sent out batched under the Data-Topic-Queue defined in the transportSettings (`UA_BrokerDataSetWriterTransportDataType.queueName`) of the WriterGroup. The reason for the implementation is, that my use-case gets simplified, if I receive DataSetMessages of multiple DataSetWriter grouped together through batching. Example JSONs and Topics of this can be found here (https://github.com/umati/Showcase/blob/pubsub/PubSub.md#topic-structure).